src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -27,18 +26,21 @@ public class ClusterTransactionDataCommand
             public CommandType Type;
             public string Id;
             public BlittableJsonReaderObject Document;
-            public LazyStringValue ChangeVector;
-            public bool IdPrefixed;
+            public string ChangeVector;
             public long Index;
 
             public static ClusterTransactionDataCommand FromCommandData(BatchRequestParser.CommandData command)
             {
+                if (command.IdPrefixed)
+                {
+                    throw new NotSupportedException("Deleting by prefix is not supported.");
```

Comment:
In what context? "deleting by prefix is not supported on cluster transaction... "

Reply:
done.